client/starwhale/core/dataset/view.py

```diff
@@ -96,3 +94,53 @@ def tag(self, tags: t.List[str], remove: bool = False, quiet: bool = False) -> N
         else:
             console.print(f":surfer: add tags {tags} @ {self.uri}...")
             self.dataset.add_tags(tags, quiet)
+
+
+class DatasetTermViewRich(DatasetTermView):
+    @classmethod
+    @BaseTermView._pager
+    @BaseTermView._header
+    def list(
+        cls,
+        project_uri: str = "",
+        fullname: bool = False,
+        show_removed: bool = False,
+        page: int = DEFAULT_PAGE_IDX,
+        size: int = DEFAULT_PAGE_SIZE,
+    ) -> t.Tuple[t.List[t.Dict[str, t.Any]], t.Dict[str, t.Any]]:
+        _datasets, _pager = super().list(
+            project_uri, fullname, show_removed, page, size
+        )
+        custom_column: t.Dict[str, t.Callable[[t.Any], str]] = {
+            "tags": lambda x: ",".join(x),
+            "size": lambda x: pretty_bytes(x),
+            "runtime": cls.place_holder_for_empty(),
+        }
+
+        cls.print_table("Dataset List", _datasets, custom_column=custom_column)
+        return _datasets, _pager
+
+
+class DatasetTermViewJson(DatasetTermView):
+    @classmethod
+    def list(
+        cls,
+        project_uri: str = "",
+        fullname: bool = False,
+        show_removed: bool = False,
+        page: int = DEFAULT_PAGE_IDX,
+        size: int = DEFAULT_PAGE_SIZE,
+    ) -> None:
+        _datasets, _pager = super().list(
+            project_uri, fullname, show_removed, page, size
+        )
+        cls.pretty_json(_datasets)
+
+    def info(self, fullname: bool = False) -> None:
+        self.pretty_json(self.get_info_data(self.dataset.info(), fullname))
+
+
+def get_term_view(ctx_obj: t.Dict) -> t.Type[DatasetTermView]:
+    if "output" in ctx_obj and ctx_obj["output"] == "json":
```

Comment:
`if "output" in ctx_obj and ctx_obj["output"] == "json"` -> `ctx_obj.get('output') == "json"` ?